docs/src/installation/deployment.md

```diff
@@ -574,11 +574,68 @@ If you decided to set up a dex server, you can parametrize the [metal role](http
 
 ## Bootstrapping a Partition
 
-Documentation will follow soon. Please come to our Slack channel when you have come to this point and require any help.
+### Management network
 
-## Partition Deployment
+To be able to deploy and maintain a metal-stack partition, you need to bootstrap the management network first. 
```

Comment:
For us the term "management network" is very clear but on Google I can't find a proper definition of what it really it is. Maybe one definition for this would be good.

Reply:
Maybe Out-Of-Band-Network is a better term.

Reply:
Yes!